### ISSUE TITLE
Cannot switch NVDA into navigator mode in Windows IDE

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5912]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.679
    OS Version      : Windows-10 1909
    R Version       : 3.5.0

### Steps to reproduce the problem
- install NVDA screen reader https://www.nvaccess.org/download/
- start it (Ctrl+Alt+N)
- install Focus Highlight addon which will show where both keyboard focus and NVDA navigator object are located https://addons.nvda-project.org/addons/focusHighlight.en.html
- start RStudio desktop IDE, ensure focus is in Console via <kbd>Ctrl+2</kbd>
- notice the blue dotted line rectangle indicating NVDA is in focus mode (aka forms mode)
- try to switch NVDA into browse mode so you can navigate the UI without moving keyboard focus <kbd>Insert+Spacebar</kbd>

### Describe the problem in detail
- nothing happens, still see the blue dotted rectangle

### Describe the behavior you expected
- should hear a sound and the rectangle should change to solid thick red indicating browse mode, and that the navigator object and keyboard focus are in the same location
- then able to move the navigator around, e.g. <kbd>Insert+Shift+Up</kbd> to move to parent
- these things work in RStudio Server in FireFox using NVDA
- this greatly reduces ability to navigate the RStudio desktop UI with NVDA; any areas we haven't fixed ability to navigate to with Tab / Shift+Tab cannot be reached

### Additional Info
- this may be another QtWebEngine issue; I will update this once I have tried to repro with a Qt example program

**Comment by gtritchie** [comment]
Also repros with QtWebEngine SimpleBrowser example, so moving out of 1.3 as unlikely we'll be able to do anything about it short of getting a fix into Qt. Also, possible this is an NVDA thing; need to look and see how it determines if it's dealing with a web page (it does work with Electron, so either NVDA or Electron is doing something to make that happen).

**Comment by gtritchie** [comment]
Related issue in NVDA repo: https://github.com/nvaccess/nvda/issues/10838

**Comment by gtritchie** [comment]
This may not be solvable while we're using QtWebEngine, see the NVDA issue for more details (nvaccess/nvda#10838).

**Comment by gtritchie** [comment]
Verified that this is working with RStudio Electron. Used NVDA 2021.3.5 with Focus Highlight addon, on Windows-11, with Electron build 2022.06.0-daily+249. Was able to switch out of focus mode into browse mode and move around the UI.
